app/models/google_event.rb

```diff
@@ -100,7 +104,7 @@ def new_time_low(time)
 
     def new_time_high(time)
       if time > time.beginning_of_hour + GRANULARITY
-        time.beginning_of_hour + 2 * GRANULARITY
+        time.beginning_of_hour + GRANULARITY + GRANULARITY
```

Comment:
Because GRANULARITY was transformed into integer and instead of 1 hour, 10 years was added.